docs/dev/tutorials/aa-daily-spend-limit.md

```diff
@@ -41,10 +39,39 @@ The current version of `zksync-web3` uses `ethers v5.7.x` as a peer dependency.
 Additionally, please install a few packages that allow us to utilize the [zkSync smart contracts](../developer-guides/system-contracts.md).
 
 ```shell
-yarn add @matterlabs/zksync-contracts @openzeppelin/contracts @openzeppelin/contracts-upgradeable
+yarn add -D @matterlabs/zksync-contracts @openzeppelin/contracts @openzeppelin/contracts-upgradeable
 ```
 
-Lastly, create `hardhat.config.ts` config file and the `contracts` and `deploy` folders like in the [quickstart tutorial](../building-on-zksync/hello-world.md).
+Also, create the `hardhat.config.ts` config file, `contracts` and `deploy` folders, similar to the [quickstart tutorial](../developer-guides/hello-world.md). In this project our contracts will interact with system contracts, to achieve that, we need to include the `isSystem: true` in the compiler settings:
+
+```typescript
+import "@matterlabs/hardhat-zksync-deploy";
+import "@matterlabs/hardhat-zksync-solc";
+module.exports = {
+    zksolc: {
+        version: "1.3.6",
+        compilerSource: "binary",
+        settings: { 
+            optimizer: { 
+                enabled: true, 
+                mode: "z", 
```

Comment:
I think 1.3.6 with optimizer options is not released yet as it's not fully documented/tested